odoo/addons/base/models/ir_fields.py

```diff
@@ -604,7 +606,7 @@ def _str_to_one2many(self, model, field, records):
         def log(f, exception):
             if not isinstance(exception, Warning):
                 current_field_name = self.env[field.comodel_name]._fields[f].string
-                arg0 = exception.args[0] % {'field': '%(field)s/' + current_field_name}
+                arg0 = sanitize(exception.args[0]) % {'field': '%(field)s/' + current_field_name}
```

Comment:
This misses the point entirely, and breaks the feature.

Reply:
I'm really not sure how to fix this with this spaghetti code